Controller/Callback/Offsite.php

```diff
@@ -210,6 +211,9 @@ public function execute()
                     case Rabbitlinepay::CODE:
                         $dispPaymentMethod = "Rabbit LINE Pay";
                         break;
+                    case Ocbcpao::CODE:
+                        $dispPaymentMethod = "OCBC PayAnyone";
```

Comment:
Should there be a space `OCBC Pay Anyone`?

Reply:
Nice catch !! -> https://github.com/omise/omise-magento/pull/341/commits/b97b2656591128b0ba7f36ca6fbb20109bd12c9f Fixed 🙏